x/poolmanager/types/pool.go

```diff
@@ -2,7 +2,7 @@ package types
 
 import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	proto "github.com/cosmos/gogoproto/proto"
+	proto "github.com/gogo/protobuf/proto"
```

Comment:
Just curious, what's the reason we're changing this from cosmos' gogo proto to the original one?

Reply:
Correct, we need to switch this back, good catch (was likely merge conflict)

Reply:
done